### DISCUSSION TITLE
caprover postgres

### DISCUSSION THREAD

**Discussion by theerudito** [caprover/caprover#1971] [General]
I have a webapi image in NET 8.0. creates a container from the image with port 8080 It works without problems in Docker Dekstop the database is in a caprover container in caprover it works fine but when uploading the application to caprover I can see swagger without problem but when doing a get it does not bring anything from the database

in localhost 
![image](https://github.com/caprover/caprover/assets/81595290/ba7e7a84-e0de-4b35-8eb8-d800d1d46b9d)

in caprover the same proyect 
![image](https://github.com/caprover/caprover/assets/81595290/1cfd79ce-8c07-49e3-94fe-bed082320ce9)

**Comment by githubsaturn** [comment]
These are your internal app errors. You need to find out what they underlying issue is. For example, are you being connected to the correct database? Are you using the correct database connection strings and credentials in CapRover compared to your local host etc etc...
